core/types/governance_new_spot_market.go

```diff
@@ -316,6 +316,10 @@ func (i InstrumentConfigurationSpot) String() string {
 	)
 }
 
+func (i InstrumentConfigurationSpot) QuoteName() string {
+	return i.Spot.QuoteAsset
```

Comment:
I don't think that's correct. Assuming the meaning is the name (vs ID of the asset). just return the i.Spot.Name

Reply:
fixed